### PR TITLE
Update meshAttributes.xml

### PR DESCRIPTION
**Pull request by daisy20170101**
fault dynamic condition can be tagged either 3 or n>65

### CODE REVIEW

**Comment by Thomas-Ulrich** [comment]
Well actually, this boundaryCondition can be used to specify any boundary condition, not only DR boundary condition.
If you want you can use boundaryCondition for setting up the free surface. (actually, when generating fully coupled meshes, we used that to set boundaryCondition=2 on the sea surface).
So I'm not sure this needs to be updated.

**Comment by daisy20170101** [comment]
I see. But 7 or 8 is not referred to any type yet, which might be confusing.

File: XmlExample/meshAttributes.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <freeSurface>1</freeSurface>
 <dynamicRupture>2</dynamicRupture>
 <absorbing>3,4</absorbing>
-<boundaryCondition tag="7">7</boundaryCondition>
-<boundaryCondition tag="8">8,9</boundaryCondition>
+<boundaryCondition tag="65">7</boundaryCondition>
```
Comment (Thomas-Ulrich) [inline XmlExample/meshAttributes.xml:4]: then maybe add: 
```<comment>see https://seissol.readthedocs.io/en/latest/fault-tagging.html for SeisSol convention regarding boundaryCondition</comment>```

Comment (krenzland) [reply XmlExample/meshAttributes.xml:4]: Why this comment format and not the standard xml-comment:

```xml
<!-- see https://seissol.readthedocs.io/en/latest/fault-tagging.html for SeisSol convention regarding boundaryCondition -->
<boundaryCondition tag="65">7</boundaryCondition> 
```
With this, you get nice syntax highlighting :)

Comment (Thomas-Ulrich) [reply XmlExample/meshAttributes.xml:4]: ha yes, that's better thanks!

**Comment by Thomas-Ulrich** [comment]
I close as the proposed changes have been integrated in #41.

**Pull request closed without merging**